ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py

```diff
@@ -1252,3 +1252,45 @@ def test_torch_multi_dot(
         test_values=True,
         tensors=x,
     )
+
+@handle_frontend_test(
+    fn_tree="torch.linalg.ldl_factor_ex",
+    dtype_and_input=helpers.dtype_and_values(
+    available_dtypes=helpers.get_dtypes("float")),
+)
+import numpy as np
+#Size of parameter has been set as 3 and could be altered to alter results 
+def generate_hermitian_matrix():
+    size = 3  # Size is predetermined as 3x3
+    # Generate a random complex matrix
+    matrix = np.random.rand(size, size) + 1j * np.random.rand(size, size)
```

Comment:
We're not supposed to generate data using np random, we have to use hypothesis, feel free to read the docs to get an idea how that should be done.